### ISSUE TITLE
gsk loading issue

### ISSUE THREAD

**Issue by zhehengluoK** [ZubinGou/math-evaluation-harness#2]
suggest change to ```dataset = load_dataset(data_name, "main", split=split)``` as the dataset changes in huggingface.
https://github.com/ZubinGou/math-evaluation-harness/blob/db77bc6bb2a17c90ed2d63e88ebd8c40a82858c5/data_loader.py#L20

**Comment by YiandLi** [comment]
Yes, there may be `Please pick one among the available configs: ['main', 'socratic']`
